algorithm_config.yaml

```diff
@@ -53,15 +53,26 @@ inputs:
       required: false
       default: all
     - name: limit
-      description: Maximum number of GEDI granule data files to download from the CMR,
-        regardless of the number of granules within the AOI.
+      description: Maximum number of GEDI granules to subset, regardless of the number
+        of granules within the spatio-temporal range.
       required: false
-      default: "1000"
+      default: "100_000"
     - name: output
       description: Name of the output file produced by the algorithm.  Defaults to
         using the AOI file name (without the extension) with the suffix "_subset.gpkg".
       required: false
       default: ""
+    - name: s3fs_open_kwargs
```

Comment:
How do we indicate to users that this arg and the proccesors args are for Advanced use only, and we don't expect them to be used normally?

Reply:
I can add a bit more detail to the docs to make it abundantly clear that these values are for advanced usage only, and that they should likely never be set.